### PR TITLE
UI integration test case: Advertise only API feature

### PR DESCRIPTION
**Pull request by Kirishikesan**
UI integration test for the following feature: [Advertise only API (Third party API). ](https://github.com/wso2/product-apim/issues/10333)

This test contains,
1. Creating a third party api and publishing it
2. Checking if AsyncAPI has other protocols option enabled
3. Checking if all the necessary changes of the third party api feature is enabled
4. Checking if necessary prompts and labels are visible
5. Checking if the authorization header and value can be customized

### CODE REVIEW

**Comment by Kirishikesan** [comment]
Fixed the issue : [Original Dev Portal link was not working](https://github.com/wso2/api-manager/issues/150)

File: tests/cypress/support/commands.js [hunk 1]
```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
...
+Cypress.Commands.add('publishThirdPartyApi', (apiName = null) => {
+
+    cy.visit(`${Utils.getAppOrigin()}/publisher/apis`);
+
+    //check if other protocol option is added in AsyncApi
+    cy.get('#itest-streaming-api-create-menu', { timeout: 30000 });
+    cy.get('#itest-streaming-api-create-menu').click();
+    cy.get('#itest-id-create-streaming-api-import').click();
+    cy.get('#outlined-full-width').type('https://raw.githubusercontent.com/asyncapi/spec/v2.0.0/examples/2.0.0/streetlights.yml');
+    cy.get('#outlined-full-width').should('have.value','https://raw.githubusercontent.com/asyncapi/spec/v2.0.0/examples/2.0.0/streetlights.yml');
```
Comment (lasanthaS) [inline tests/cypress/support/commands.js:371]: Line length should be 120 chars maximum.

File: tests/cypress/support/commands.js [hunk 2]
```diff
@@ -359,3 +359,103 @@ Cypress.Commands.add('logoutFromPublisher', () => {
...
+    cy.get('[data-testid="itest-no-tier-dialog"]').contains('No tiers are available for the API.').should('exist');
+
+    //Check if authorization header and value can be customized
+    cy.get('#left-menu-test').click();
+    cy.get('#advAuthHeader').should('exist');
+    cy.get('#advAuthHeaderValue').should('exist');
+
+})
+
+
```
Comment (lasanthaS) [inline tests/cypress/support/commands.js:461]: Remove additional new lines. Only 1 new line would be enough.

File: tests/cypress/support/commands.js (hunk 1 above)
Comment (Kirishikesan) [reply tests/cypress/support/commands.js:371]: This is fixed in [#314](https://github.com/wso2/apim-apps/pull/314)

File: tests/cypress/support/commands.js (hunk 2 above)
Comment (Kirishikesan) [reply tests/cypress/support/commands.js:461]: This is fixed in [#380](https://github.com/wso2/apim-apps/pull/380)

**Pull request merged**